packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -175,6 +190,16 @@ export function useGridCell<T, C extends GridCollection<T>>(props: GridCellProps
           );
         }
         break;
+      case 'Tab':
+        // Track if focus is being moved around within the grid via alt/option + tab so we can update focusedKey
+        // when onFocus is called for the next focusable grid cell.
+        if (e.altKey) {
+          isAltTabEvent = true;
+          // Set the grid cell's tab index to -1 so that shift+alt/option+tab doesn't move focus from the focusable child
+          // to the grid cell. This would cause the child to then get refocused, effectively nullifying your shift+alt+tab action
+          ref.current.tabIndex = -1;
+        }
+        break;
```

Comment:
Open to other ideas that don't need this `isAltTabEvent` tracker. Problem is that the keyboard delegate doesn't have a way of doing "nextFocusableKey" or "previousFocusableKey" so the approach used in useSelectableCollection is out.